tests/integration_tests/python/test_download.py

```diff
@@ -55,10 +60,13 @@ def test_download(data_builder, file_form, as_admin, api_db):
 
     # Verify a single file in tar with correct file name
     found_second_session = False
+    found_third_session = False
     for tarinfo in tar:
         assert os.path.basename(tarinfo.name) == file_name
         if 'session1_0' in str(tarinfo.name):
             found_second_session = True
+        if 'session1_1' in str(tarinfo.name):
+            found_third_session = True
     assert found_second_session
 
```

Comment:
Should found_third_session be tested here after the loop?

Reply:
Whoops - nice catch!